theme/templates/resource-landing-page/title-section.html

```diff
@@ -402,20 +402,30 @@ <h2 id="resource-title">{{ title }}</h2>
 	},
 	"license": {
 	    "@type": "CreativeWork",
-	    "text": "{{ cm.metadata.rights.statement }}",
 	    {% if cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution CC BY." %}
+	    "text": "{{ cm.metadata.rights.statement }}",
         "url": "http://creativecommons.org/licenses/by/4.0/"
         {% elif cm.metadata.rights.statement == "This resource is shared under the Creative Commons Attribution-ShareAlike CC BY-SA." %}
+        "text": "{{ cm.metadata.rights.statement }}",
```

Comment:
Why check rights statement and use a url depending on the rights statement?  Why not just use the statement and url that is in the rights object?